chunky/src/java/se/llbit/chunky/chunk/BlockPalette.java

```diff
@@ -130,6 +131,30 @@ public int put(BlockSpec spec) {
     }
   }
 
+  private void putLegacyTags() {
+    byte[] block = new byte[1];
+    byte[] data = new byte[1];
+    for (int i = 0; i < 256; i++) {
+      for (int j = 0; j < 16; j++) {
+        block[0] = (byte) i;
+        data[0] = (byte) j;
+        putLegacy(LegacyBlocks.getTag(0, block, data));
+      }
+    }
+  }
+
+  private void putLegacy(Tag tag) {
+    BlockSpec spec = new BlockSpec(tag);
+    blockMap.computeIfAbsent(spec, k -> palette.size());
+    try {
+      Block block = spec.toBlock();
+      applyMaterial(block);
+      palette.add(block);
+    } catch (Throwable t) {
+      palette.add(null);
```

Comment:
This should never happen; if it does, we should log an error and put Air into that index instead of `null` (which I guess will crash and burn)?

Reply:
Not true, there are sanity checks in blocks that throw a LOT (checking that a furnace isn't facing down for example)
This attempts to create every possible legacy block state, so there are bound to be such invalid states

Reply:
Could you comment that for future readers?

Reply:
At least we shouldn't catch `Throwable` but maybe introduce some `IllegalBlockStateException` or catch `RuntimeException` here for now.